src/main/java/com/weaponcharges/WeaponChargesItemOverlay.java

```diff
@@ -66,7 +69,11 @@ public void renderItemOverlay(Graphics2D graphics, int itemId, WidgetItem itemWi
 					boolean isLowCharge = charges <= chargedWeapon.getLowCharge(config);
 					if (!isLowCharge && displayWhen == DisplayWhen.LOW_CHARGE && !plugin.isShowChargesKeyIsDown()) break;
 
-					topText.setText(String.valueOf(charges));
+					if (chargedWeapon == ChargedWeapon.SERPENTINE_HELM && charges == 0) {
```

Comment:
Make this empty thing a config option that affects all weapons, replacing "0" on all uncharged versions with "Empty".

Reply:
Finished. Pushing in a moment.